controller/upload_file.rst

```diff
@@ -243,6 +243,16 @@ logic to a separate service::
 
             return $fileName;
         }
+
+        public function getTargetDir()
+        {
+            return $this->targetDir;
+        }
+
+        public function setTargetDir($targetDir = '')
```

Comment:
Fair observation. Perhaps it would be better to move default empty string to property declaration?